src/main.c

```diff
@@ -152,15 +153,35 @@ main(int argc, char* argv[])
         { "dump-config", no_argument, 0, 'D' },
         { "rotate", no_argument, 0, 'R' },
         { "config", required_argument, 0, 'f' },
+        { "no-config", required_argument, 0, 'F' },
```

Comment:
One thing I noticed here, which I got with a c&p -- it uses 'required_argument' for 'no-config' option, but it doesn't expect a value after the option flag. Should it be changed to 'no_argument'? It didn't seem to have any problem when I tested.

Thanks for merging!